includes/types/JsAny.hpp

```diff
@@ -39,8 +37,9 @@ namespace JS {
         Any() : value(JS::Undefined{}) {}
         explicit Any(int v) : value(static_cast<double>(v)) {}
         explicit Any(double v) : value(v) {}
-        explicit Any(const std::string& v) : value(v) {}
-        explicit Any(const char* v) : value(std::string(v)) {}
+        explicit Any(Rope v) : value(v) {};
```

Comment:
should be const Rope &V to make it faster

Reply:
added a constructor when we can pass address but we still need normal constructor in certain case